packages/swing-store/test/test-stateSync.js

```diff
@@ -116,10 +116,21 @@ test('export and import data for state sync', async t => {
 
   const exporter = makeSwingStoreExporter(dbDir);
 
+  const compareElems = (a, b) => a[0].localeCompare(b[0]);
```

Comment:
why `localeCompare` ?

Reply:
Because it's how you compare strings?  Is there some trick I don't know about?

It's a string comparator function that matches what `sort` wants.  It has a weird name and more power than this use case needs, but it's easier than writing my own.

Reply:
Old versions of SES use to miss those, exactly because locale related functions have too much power. It looks like it now provides a [non-locale aware implementation](https://github.com/endojs/endo/blob/c98ca1394831425cd809fef49fa19fc0f49f6da5/packages/ses/src/tame-locale-methods.js#L20-L35) so I guess it's safe to use.